### DISCUSSION TITLE
Asking to help the project.

### DISCUSSION THREAD

**Discussion by theoUniv** [FossifyOrg/General-Discussion#93] [General]
Hey, we are Théo Lemaire and Aymane Ichou.

Me and my collegue, are student in the Paris - 8 University in computer science domain.
For a school project, we need to contribute to a open source project.

We got interest for Fossify and we want to know what we can do to help the project.

We can fix issues/bug, add features that you are thinking about, improve efficiency of the project.
Translation English to French, looking for bugs or test beta to see if it's working right. We got Linux, Macos and Windows machines.

You can contact us with email:

Me: theo.lemaire03@etud.univ-paris8.fr
My collegue: aymane.ichou02@etud.univ-paris8.fr

Our discord: 
theo3378
ichou0480

**Comment by Aga-C** [comment]
If you want to help with development stuff, you can take any issue that doesn't have `needs triage` label. They are ready to be taken and done anytime by anyone who wants to contribute. Please just restrict your work only to apps that are currently released, because they won't have any major refactorings done, that could lead to merge conflicts.

In case you'd prefer to help outside the code, take a look here: https://github.com/orgs/FossifyOrg/discussions/69#discussioncomment-8009887.
